bspump/avro/deserializer.py

```diff
@@ -1,17 +1,19 @@
 #!/usr/bin/env python3
 import logging
 import io
-import bspump.avro
 import fastavro
-import bspump.trigger
+from ..avro import loader
+from .. import Generator
```

Comment:
`from ..abc import Generator`

This way you minimise the risk of circular dependencies.